examples/flask_app/README.md

```diff
@@ -0,0 +1,37 @@
+# PCF-Flask-API
+The purpose of this app is to show how to use PCF in a Flask api to spin up cloud resources. In this particular example we will utilize Capital One's Particle Cloud Framework to deploy ec2-route53 quasiparticle.
```

Comment:
the purpose of this example is to show one of multiple ways to use Particle Cloud Framework (PCF) to deploy cloud infrastructure which in this case is using a Flask api. In this particular example we deploy the ec2-route53 quasiparticle which creates a route53 record and adds the desired number on ec2 instances to the record set. to use in your environment  simply change some of the configurations parameters in the quasiparticle definition.